Contest_Control_System_Requirements.md

```diff
@@ -7,7 +7,7 @@ permalink: /ccs_system_requirements
 
 This document specifies requirements for operation, verification and
 validation of Programming Contest Control Systems that wish to be
-considered for managing the operation of the [ACM International
+considered for managing the operation of the [International
 Collegiate Programming Contest](http://icpc.baylor.edu) [World
```

Comment:
Link should now be https://icpc.global

Reply:
Sorry, missed this. I will fix via a future PR.